apps/docs/src/guide/contracts/snippets/proxy-contracts.ts

```diff
@@ -0,0 +1,105 @@
+// #region proxy-2
+import {
+  Provider,
+  Wallet,
+  Src14OwnedProxy,
+  Src14OwnedProxyFactory,
+} from 'fuels';
+
+import { LOCAL_NETWORK_URL, WALLET_PVT_KEY } from '../../../env';
+import {
+  Counter,
+  CounterFactory,
+  CounterV2,
+  CounterV2Factory,
+} from '../../../typegend';
+
+const provider = await Provider.create(LOCAL_NETWORK_URL);
+const wallet = Wallet.fromPrivateKey(WALLET_PVT_KEY, provider);
+
+const counterContractFactory = new CounterFactory(wallet);
+const deploy = await counterContractFactory.deploy();
+const { contract: counterContract } = await deploy.waitForResult();
+// #endregion proxy-2
+
+// #region proxy-3
+/**
+ * It is important to pass all storage slots to the proxy in order to
+ * initialize the storage slots.
+ */
+const storageSlots = Counter.storageSlots.concat(Src14OwnedProxy.storageSlots);
+
```

Comment:
@danielbate Perhaps it would be clearer to retrieve the `storageSlots` directly from the instantiated factory. It seems to make the flow a bit clearer. What do you think?

```suggestion
const storageSlots = counterContractFactory.storageSlots.concat(Src14OwnedProxy.storageSlots);
```